.babelrc

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
**TODO FOR LOST POINTS**

Just putting a top-level comment that you don't have a test folder with any test modules and you need those for this lab.